cli/cli/commands/service/service_helpers/service_helpers.go

```diff
@@ -0,0 +1,355 @@
+package service_helpers
+
+import (
+	"context"
+	"fmt"
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/kurtosis_core_rpc_api_bindings"
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/enclaves"
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/services"
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/starlark_run_config"
+	"github.com/kurtosis-tech/kurtosis/api/golang/engine/kurtosis_engine_rpc_api_bindings"
+	"github.com/kurtosis-tech/kurtosis/api/golang/engine/lib/kurtosis_context"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/helpers/user_services"
+	"github.com/kurtosis-tech/stacktrace"
+	"strconv"
+	"strings"
+)
+
+const (
```

Comment:
this is just pulling consts and functions that are now shared by `service add`, `service update`, and `service inspect`